test/acceptance/www.hurl

```diff
@@ -0,0 +1,4 @@
+GET {{proto}}://www.{{host}}/
+HTTP 301 # expect moved permanently response
+[Asserts]
+header "Location" == "https://{{host}}/"
```

Comment:
This seems to be testing that `www.` redirects to the apex. That's fine. But it seems under-specified. Do you want `https://www.{{host}}/uploads/podcast/news-2022-06-27/the-changelog-news-2022-06-27.mp3` to redirect to `https://www.{{host}}/uploads/news/1/changelog-news-1.mp3` or to `https://{{host}}/uploads/news/1/changelog-news-1.mp3`?

The answer to that determines where and when you do this redirect.

Reply:
As long as you're OK with multiple redirects: (1) `www.` to apex, then (2) old news URL to new.

Reply:
I **think** that's fine considering that these will be hit rarely. To be honest, I want to double-check with @jerodsanto why we need these news mp3 redirects. The original URLs work, so this must be done to keep things consistent. We are squarely in the **make it work** phase now, I'm thinking make it better & faster in follow-ups so that we ship 1.0-rc.1 and start routing some production traffic through it 🧑‍🔧